tests/unit/test_specs.py

```diff
@@ -162,3 +181,27 @@ def test_validate_supported_bundles_fail():
         specs.validate_supported_bundles(TEST_ITEM_TYPE,
                                          'analytic',
                                          ALL_PRODUCT_BUNDLES)
+
+
+def test_get_supported_assets_success():
```

Comment:
this test is okay as-is. I am just concerned with the cost of maintaining a static list it in light of spec changes. You could minimize the overhead by testing that a specific asset, e.g. `basic_udm2` (since it's already being used in the tests) is in the list of returned assets. that way you confirm that the supported assets were retrieved successfully and have reasonable confidence in the quality of the return (though, admittedly, not total confidence) but the test doesn't have to be updated with any additions to the supported assets. Balancing some uncertainty with cost of maintaining the code is a dilemma in testing. It's what makes this art.

Reply:
🎨

Reply:
I do agree with balancing confidence in tests with maintainability. Though, this makes me wonder if we should be doing this with all tests that query `ALL_PRODUCT_BUNDLES` and `ALL_ITEM_TYPES`. Surely, these lists will be dynamic.

Reply:
That being said, I'll happily follow your suggestion and just assert `TEST_ASSET_TYPE in supported_assets`